egs/mgb2/asr/conf/train_pytorch_transformer_large_ngpu4.yaml

```diff
@@ -0,0 +1,27 @@
+# This configuration requires 4 gpus with 12GB memory
```

Comment:
You can move this file to `egs/mgb2/asr/conf/tuning/` and add a symbolic link like https://github.com/espnet/espnet/blob/master/egs/librispeech/asr1/conf/train.yaml

Reply:
done, I also updated the run.sh accordingly.